src/types.ts

```diff
@@ -1,18 +1,18 @@
-import {UserMethods} from './users';
+import {UserApi} from './users';
 
 export interface SdkConfig {
     token?: string;
     baseUrl?: string;
 }
 
-export type JSONRequest = (
+export type JSONRequest<T = {}> = (
```

Comment:
Actually it looks like sine this has a default, we could probably leave it off and it should work in most of the cases.

Reply:
This is what happens when there's no default...

![](https://d.pr/i/zvWU3C+)